src/lektorium/app.py

```diff
@@ -0,0 +1,103 @@
+import enum
+import functools
+import logging
+import sys
+import aiohttp.web
+import aiohttp_graphql
+import bs4
+import graphene
+from . import schema
+from . import install as client
+
+
+async def index(request, app_path, auth0_options=None):
+    index = app_path / 'index.html'
+    data = index.resolve().read_bytes()
+    data = bs4.BeautifulSoup(data, 'html.parser')
+    if auth0_options is not None:
+        for k, v in auth0_options.items():
+            data.find('body')[k] = v
+    return aiohttp.web.Response(
+        body=str(data).encode('utf-8'),
+        content_type='text/html',
+    )
+
+
+class RepoType(enum.Enum):
+    LIST = enum.auto()
+    LOCAL_FAKE = enum.auto()
+    LOCAL_ASYNC = enum.auto()
+    GIT = enum.auto()
+
+
+def create_app(repo_type=RepoType.LIST, auth=''):
+    from . import repo
+    if repo_type == RepoType.LIST:
+        repo = repo.ListRepo(repo.SITES)
+    elif repo_type in (RepoType.LOCAL_FAKE, RepoType.LOCAL_ASYNC):
+        if repo_type == RepoType.LOCAL_FAKE:
+            from .repo.local import FakeServer
+            server = FakeServer
+        else:
+            from .repo.local import AsyncLocalServer, LocalLektor
+            server = AsyncLocalServer()
+        repo = repo.LocalRepo('gitlab', server, LocalLektor)
```

Comment:
Is `LocalLektor` available here in case of `RepoType.LOCAL_FAKE`?

Reply:
Yeah, Max already fixed it I hope.